elements/bulbs-truncate-element/bulbs-truncate-element.scss

```diff
@@ -0,0 +1,27 @@
+.bulbs-truncate-element-parent-active {
+  height: 1000px;
+  overflow: hidden;
+  position: relative;
+
+  &::after {
+    background-image: linear-gradient(to bottom, rgba(255, 255, 255, 0) 0%, rgba(255, 255, 255, 1) 80%);
+    bottom: 0;
+    content: "";
+    display: block;
+    height: 300px;
+    left: 0;
+    position: absolute;
+    right: 0;
+    top: auto;
+  }
+}
+
+.bulbs-truncate-element-button {
```

Comment:
could add `outline: none` here to prevent that annoying blue border when clicking